_data/places.yml

```diff
@@ -1,35 +1,54 @@
+# - name: Example Eatery
+#   status: open
+#   phone: (212) 555-1234
+#   url: https://www.example.com/
+#   instagram: https://www.instagram.com/
+#   hours: 9AM – 5PM (Mon – Fri)
+#   notes: "Example text here"
+#   delivery:
+#     uber_eats:
+#     doordash:
+#     postmates:
+#     seamless:
+#     caviar: 
+#     toast:
+#   links:
+#     - title: Staff relief fund
+#       url: https://www.gofundme.com/
+#   last_updated: 5/7
+
 - name: Acme Pet Food Inc
   instagram: https://www.instagram.com/acmepetfood/
   phone: (718) 789-8062
-  hours: 10AM - 7PM (Mon - Fri), 10AM - 5PM (Sat), 11AM - 5PM (Sun)
+  hours: 10AM – 7PM (Mon – Fri), 10AM – 5PM (Sat), 11AM – 5PM (Sun)
   notes: “Limit exposure to our staff and yourself, please wear a mask when visiting, wait outside if crowded. No contact curbside and local home delivery available.”
   last_updated: 4/13
 
 - name: Alta Calidad
   status: closed
   url: https://altacalidadbk.com
   instagram: https://www.instagram.com/altacalidadbk/
-  hours: 11AM - 8PM⁣
+  hours: 11AM – 8PM⁣
   delivery:
     uber_eats: https://www.ubereats.com/new-york/food-delivery/alta-calidad/L0ktkVRjQbOkllv7UWe1fQ
     doordash: https://www.doordash.com/store/alta-calidad-brooklyn-133126/
     postmates: https://postmates.com/merchant/alta-calidad-brooklyn
     seamless: https://www.seamless.com/menu/alta-calidad-552-vanderbilt-ave-brooklyn/460349
     caviar: https://www.trycaviar.com/m/alta-calidad-5914
     toast: https://www.toasttab.com/alta-calidad/v3
-  notes: “Just in time for #TacoTuesday, #TakeoutTuesday and #CincodeMayo, we’ll be back with contactless pickup on 5/5 from 11am-8pm! Head to our website to preorder your favorite tacos, guac and batched margaritas, and we’ll have it all ready and waiting on our patio at your designated pickup time on Tuesday. 🌮🥳🍹”
+  notes: Launching Alta Calidad Mercado on May 13. Will sell prepared foods, specialty goods and pantry items.
```

Comment:
Oh nice! Is there a source for this? Would like to link to it from the newsletter

Reply:
Just the text on their homepage, though I haven’t signed up for their newsletter yet: https://altacalidadbk.com